src/oc_erchef/apps/chef_objects/priv/depselector_rb/Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
 
-gem 'dep_selector', :git => 'https://github.com/chef/dep-selector', :branch => 'master'
-gem 'erlectricity', :git => 'https://github.com/sdelano/erlectricity', :branch => 'master'
+gem 'dep_selector', git: 'https://github.com/chef/dep-selector', branch: 'master'
+gem 'erlectricity', git: 'https://github.com/chef/erlectricity', branch: 'modernize'
```

Comment:
@phiggins You're right, I think. Also, I don't believe anyone would be using that repo, so... if you'd like to, please feel free to adopt the PR 😃